spec/github_helper_spec.rb

```diff
@@ -418,7 +418,8 @@ def create_release(assets: [])
         target: test_target,
         description: test_description,
         assets: assets,
-        prerelease: false
+        prerelease: false,
+        is_draft: true
```

Comment:
🧪  I see you added the `is_draft` parameter to the call site, but not yet added any new Unit Tests to validate that different values of `is_draft` in the calls to `GithubHelper#create_release` will indeed result in `client.create_release` and GitHub API calls that contains the proper `draft` parameter too.

Would you like to give it a try? I think that, by taking inspiration from some of the existing tests, adding 2 small tests (one to test the case of `is_draft: true` and another for `is_draft: false`) to ensure that passing those values will indeed result in the `client` be called with the expected parameter… too shouldn't be too difficult, but let me know if you need some help with that.

Reply:
I've added the unit test in 173f1c264167fbf54573a6f2e0d691c0ae3a29a9. I'm relatively new to Ruby's world of unit testing. So, please feel free to point out any mistakes or if I can make any additional improvements.

Reply:
Those tests seems good to me 👍  Great job!

_PS: In an ideal word, we'd not only add tests for the internal helper class (`github_helper.rb`) like there, but also for the `CreateReleaseAction` Fastlane action (which is what is exposed as public action for consumption when users call `create_release` from their Fastfile). But we have zero tests for this action at the moment (only tests for the helper it wraps), so maybe this PR is not the place to start adding a brand new `create_release_action_spec.rb` file and tests to that action, and we'll keep that for later (though… writing _those_ action unit tests would have made you notice that you forgot to use the new `ConfigItem` in the action's implementation, so… 😅 )_